src/pint/scripts/pintk.py

```diff
@@ -160,10 +160,11 @@ def main(argv=None):
     args = parser.parse_args(argv)
 
     root = tk.Tk()
+    root.minsize(800, 600)
     if not args.test:
         app = PINTk(root, parfile=args.parfile, timfile=args.timfile, ephem=args.ephem)
         root.protocol("WM_DELETE_WINDOW", root.destroy)
-        root.mainloop()
+        tk.mainloop()
```

Comment:
Any idea why this needed to change from `root` to `tk`?  Seems strange since the other change just modified `root`....

Reply:
I don't think it is needed, but the examples I saw all did it that way, so I changed it. Probably it is synonymous.